src/pykeen/nn/node_piece/anchor_search.py

```diff
@@ -10,9 +10,11 @@
 import scipy.sparse
 import torch
 from class_resolver import ClassResolver, OptionalKwargs
+from torch_ppr import page_rank
+from torch_ppr.utils import edge_index_to_sparse_matrix, prepare_page_rank_adjacency, prepare_x0
 from tqdm.auto import tqdm
 
-from .utils import edge_index_to_sparse_matrix, page_rank, prepare_page_rank_adjacency
+from .utils import ensure_num_entities
```

Comment:
this might not exist yet

Reply:
https://github.com/pykeen/pykeen/pull/995/commits/d2ed47010d80dbf4b273a35e4a9b3e00af1f1122